roles/translator/src/main.rs

```diff
@@ -35,8 +37,10 @@ fn process_cli_args() -> ProxyResult<ProxyConfig> {
 
 #[async_std::main]
 async fn main() {
+    tracing_subscriber::fmt::init();
+
     let proxy_config = process_cli_args().unwrap();
-    println!("PC: {:?}", &proxy_config);
+    warn!("PC: {:?}", &proxy_config);
```

Comment:
Think this should just be `info` instead of `warn`

Reply:
Yeah - I agree! I just tried to keep it printing always like it was - maybe I should have taken some artistic liberties!